compiler-plugin/build.gradle

```diff
@@ -1,5 +1,8 @@
+plugins {
+    id 'java-gradle-plugin'
```

Comment:
Why do we need those plugins? :)

Reply:
Yeah, java-gradle-plugin is not needed, but maven-plugin is required for local maven deployment

Reply:
Nop it is not.